.gitignore

```diff
@@ -16,3 +16,5 @@
 /polyfills/Intl/~locale
 
 polyfills/Promise/polyfill.js
+
+polyfills/atob/polyfill.js
```

Comment:
Line spacing here seems a bit strange.

Is it possible to have .gitignore files at sublevels within the repo?  If so we could put it in the polyfill directory.  Alternatively, we could have a different name for polyfill files that are auto-installed, so that they can be ignored with a single pattern.  But that's probably unnecessary.